最小限度版のREADME作成しました。

fsinakanoko opened the pull request:
README作成完了しました。
Arduino技術者の方が想定読者のため、Arudino IDEの設定はURLの紹介のみにしています。
内容の過不足等確認お願いします。

nakanokofsi committed:
新規作成

nakanokofsi committed:
誤字修正

OmronIXI commented:
ありがとうございます。
基本的な内容はばっちりですが、以下をご対応ください。
・カバン型の環境センサについて基本的な説明を最初に追加ください
・サンプルプログラムが何の情報をどのように取得できるかの概要を記述ください
・スケッチプログラム→スケッチ
・HardWare→Hardware

nakanokofsi committed:
指摘事項に合わせて修正

■指摘事項一覧
・カバン型の環境センサについて基本的な説明を最初に追加ください
・サンプルプログラムが何の情報をどのように取得できるかの概要を記述ください
・スケッチプログラム→スケッチ
・HardWare→Hardware
